terraform/staging/main.tf

```diff
@@ -7,10 +7,6 @@ provider "aws" {
 }
 data "aws_caller_identity" "current" {}
 data "aws_region" "current" {}
-locals {
```

Comment:
Why are we removing the `locals`?

Reply:
I'm not too sure, I think louise removed it, but not for the reason, don't see a documentation on it too

Reply:
Cool. 👍 There might be a reason it was removed. Not sure either.